.github/workflows/pr.yml

```diff
@@ -1,27 +1,89 @@
 name: Build CI
 
 on:
+  push:
```

Comment:
this doubles action runs. master branch protected and cannot be pushed without PR.

Reply:
Without these lines there is warning
`Warning: 1 issue was detected with this workflow: Please specify an on.push hook so that Code Scanning can compare pull requests against the state of the base branch`

Reply:
This warning do not affect on checks tab. So its not a big deal

Reply:
if it required to compare with base branch. We need to make changes in ci workflow too. Either its not a big deal since it does not affect checks